build.gradle

```diff
@@ -9,7 +9,7 @@ plugins {
 
 ext {
     wordPressUtilsVersion = '2.7.0'
-    wordPressLoginVersion = 'trunk-657bed28fa735780e3ffcc214503a9e5d1286d6c'
+    wordPressLoginVersion = 'trunk-e11535105725e02dd1e8aa8065314ceb67079278'
```

Comment:
This matches the sha1 of current Login's `trunk`, so 👍 

But that Login commit… points to `fluxc:trunk-e94126bf9eb942d262768f8851cf2c4980e249fc` [in its own `build.gradle`](https://github.com/wordpress-mobile/WordPress-Login-Flow-Android/blob/e11535105725e02dd1e8aa8065314ceb67079278/WordPressLoginFlow/build.gradle#L58), as opposed to pointing to FluxC's latest commit in trunk aka `trunk-eb1d39ad095c01367af03c4259ab93cda14c675e` aka what's also used as `fluxCVersion` in this WPAndroid `build.gradle` on line 24. And FluxC's `e94126b` (which Login points to) is 3 commits behind `eb1d39a`.

While it's not surprising to see Login's pointer to fluxc be behind the FluxC we points to in the clients, especially because FluxC is updated way more often than the Login lib is, I'm a bit surprised by this given that Login and FluxC were just synchronzed a couple of hours ago — in fact, that is the whole points of this update of both Login and FluxC being done in parallel here.

So I'm wondering if this is expected to have Login not include those 3 recent commits from FluxC while this was all about synchronizing and updating the version of FluxC that both were pointing to to match 🤔

Reply:
Ok after checking with @ParaskP7 in Slack, this is in fact expected. And the 3 FluxC commits that Login does not include are not relevant to the Login library anyway.

> While it's not surprising to see Login's pointer to fluxc be behind the FluxC we points to in the clients, especially because FluxC is updated way more often than the Login lib is, I'm a bit surprised by this given that Login and FluxC were just synchronzed a couple of hours ago 

So in practice this is just because FluxC already shipped a couple of recent PRs after this synchronization in Login lib happened earlier today. So… the ship (and PRs that were pending in FluxC) is moving faster than I thought in FluxC after all 😄

Reply:
👋 @AliSoftware !

Short answer, yes, this is expected and thank you for asking! 🥇 

- This morning we fixed an issue with the [Login PR](https://github.com/wordpress-mobile/WordPress-Login-Flow-Android/pull/92) and then update WCAndroid to point to that with a [WCAndroid PR](https://github.com/woocommerce/woocommerce-android/pull/7240).
- Then, later on today, I merged 3 more PRs on FluxC ([here](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2506), [here](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2504) and [here](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2507)) and completed the project I was working for this sprint.
- This problem that this PR is trying to fix was on a previous FluxC commit, even before there extra 3 PRs, which got already merged 10 days ago on FluxC (via [this PR](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2490) and [this commit](https://github.com/wordpress-mobile/WordPress-FluxC-Android/pull/2490/commits/8a96a86fb723285d8a056231ebad6096231aadd9#diff-5c8976cc6b3f8b603138c635d67b4938b5628f9070c4030c5b0b659ba70d40d4)).
- Thus these 3 PRs that were merged afterwards weren’t going to help with this `Login` problem anyway, they are just nice to have on top of what `Login`'s `FluxC` is pointing to.

Just that makes sense to you? 🤔

Reply:
That makes total sense now, thanks for explaining in such details 👍

Reply:
Great, thank you for taking the time to investigate all that! 🥇